clusters/build-clusters/01_cluster/openshift-ingress-operator/REAME.md

```diff
@@ -0,0 +1,59 @@
+# openshift-ingress-operator
+
+## CA Certificate for app routes
+
+Openshift 4.2 has doc on [this topic](https://docs.openshift.com/container-platform/4.2/authentication/certificates/replacing-default-ingress-certificate.html).
+
+### Manual steps
+
+#### AWS IAM configuration
+
+[set up aws IAM user](https://github.com/Neilpang/acme.sh/wiki/How-to-use-Amazon-Route53-API): user `openshift-ci-robot` in group `acme` which
+    has the policy `acme`.
+
+#### Generate the certificate
+
+Using [acme.sh](https://github.com/Neilpang/acme.sh)
+
+```bash
+$ podman run -it --rm neilpang/acme.sh /bin/ash
```

Comment:
Could you provide the literal copy-paste commands or a script?

Reply:
The commands below are the recording of what I dd.
The script will be developed when periodic job is defined.